docs/thick-plugin.md

```diff
@@ -53,38 +53,62 @@ kubectl apply -f deployments/multus-daemonset-thick.yml
-`-additional-bin-dir=/opt/multus/bin`, `-multus-log-level=debug`, etc.
-
-Furthermore, it also accepts a new command line parameter, where the user
-specifies the path to the server configuration:
+The available command line parameters are:
 
 - `config`: Defaults to `"/etc/cni/net.d/multus.d/daemon-config.json"`
-- `metricsPort`: Metrics port (of multus' metric exporter), default is disable
+- `version`: Prints the daemon config version and exits
 
-### Server configuration
+### Server / Daemon configuration
 
 The server configuration is encoded in JSON, and allows the following keys:
 
 - `"chrootDir"`: Specify the directory which points to host root from the pod. See 'Chroot configuration' section for the details.
-- `"socketDir"`: Specify the location where the unix domain socket used for
-client/server communication will be located. Defaults to `"/run/multus"`.
+- `"daemonSocketDir"`: Specify the location where the unix domain socket used
+for client/server communication will be located. This is the location where the
+**Daemon** will read the configuration from. Defaults to `"/run/multus"`.
+- `"metricsPort"`: Metrics port (of multus' metric exporter); by default, no port
+is provided.
+- `"logFile"`: the path to where the daemon logs will be persisted.
+- `"logLevel"`: the logging level for the multus daemon logs.
+- `"logToStderr"`: enable this to have the daemon multus logs echoed to stderr
+as well. By default, it is disabled.
 
 In addition, you can add any configuration which is in [configuration reference](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/configuration.md#multus-cni-configuration-reference). Server configuration override multus CNI configuration (e.g. `/etc/cni/net.d/00-multus.conf`)
 
+Below you can see an example of the daemon configuration:
+```json
+{
+        "chrootDir": "/hostroot",
+        "confDir": "/host/etc/cni/net.d",
+        "logToStderr": true,
+        "logLevel": "verbose",
+        "logFile": "/tmp/multus.log",
+        "binDir": "/opt/cni/bin",
+        "cniDir": "/var/lib/cni/multus",
+        "daemonSocketDir": "/host/run/multus/",
+        "cniVersion": "0.3.1",
+        "cniConfigDir": "/host/etc/cni/net.d",
+        "multusConfigFile": "auto",
+        "multusAutoconfigDir": "/host/etc/cni/net.d"
+    }
+```
+
+### Client / Shim configuration
+
+The multus shim configuration is encoded in JSON, and essentially is just a
+regular CNI configuration, usually available in `/etc/cni/net.d/00-multus.conf`.
+
+It allows the following keys:
+
+- `"cniVersion"`: the CNI version for the Multus CNI plugin.
+- `"logFile"`:  the path to where the daemon logs will be persisted.
+- `"logLevel"`: the logging level for the multus daemon logs.
+- `"logToStderr"`: enable this to have the daemon multus logs echoed to stderr
+  as well. By default, it is disabled.
+- `"shimSocketDir"`: Specify the location where the unix domain socket used for
```

Comment:
I'm curious that "shimScoketDir" option name in multus-shim. This option specifies server's socket dir, so we may change that to 'daemonSocketDir' here? "shimSocketDir" looks like the different soket from above.

We may change 'daemonSocketDir' to 'socketDir' in multus-daemon side as well. What do  you think about that?

Reply:
I don't follow. This was intentional - it's the same socket, but located at a different path. 

Since afaiu we de-serialize the same object to both the shim config and to the daemon config, IMO, it makes sense to use different attribute names. Having the same attribute name, is, imo, confusing.